providers/aws/inputsources/composite/stackoutput.ftl

```diff
@@ -8,3 +8,41 @@
         "DeploymentUnit" : outputFilter.DeploymentUnit
     }]
 [/#function]
+
+[#macro aws_input_composite_stackoutput_seed id="" deploymentUnit="" level="" region="" account=""]
```

Comment:
Was there a plan for these params or are they just necessary for the way its called?

Reply:
They aren't required for composite stack outputs but they are used for the mock input engine. It needs the parameters so that it can provide mock responses using the details provided in the parameters